ember.md

```diff
@@ -30,36 +30,25 @@ Extract multiple values from data stored in objects and arrays.
 
 <a name="destructuring--objects"></a>
 ### 2.1 Destructuring Objects
+In Ember.js 2.16 the recommended way to access framework code in Ember applications is via the JavaScript modules AP. This makes Ember feel less overwhelming to new users, and make Ember applications start faster, by replacing the Ember global with a first-class system for importing just the parts of the framework you need.
```

Comment:
AP should be API.  Do we need the supporting sentence or  should we just direct them to ember doc